tools/solidityUpgrade/SourceUpgrade.cpp

```diff
@@ -422,7 +422,7 @@ bool SourceUpgrade::readInputFiles()
 	if (m_args.count(g_argInputFile))
 		for (string path: m_args[g_argInputFile].as<vector<string>>())
 		{
-			auto infile = boost::filesystem::path(path);
+			boost::filesystem::path infile = path;
 			if (!boost::filesystem::exists(infile))
 			{
 				if (!ignoreMissing)
```

Comment:
While updating this I noticed that `solidity-upgrade` has the file processing logic duplicated from `CommandLineInterface` and now, after the recent round of CLI refactors, it's out of sync. #11651.